cmd/virt-launcher/node-labeller/node-labeller.sh

```diff
@@ -1,6 +1,25 @@
 #!/bin/bash
 
 set -e
+PLATFORM=$(uname -m)
+case ${PLATFORM} in
+      x86_64*|i?86_64*|amd64*)
+               ARCH="x86_64"
+               MACHINE="q35"
+               ;;
+      ppc64le)
+               ARCH="ppc64le"
+               MACHINE="pseries"
+               ;;
+      aarch64*|arm64*)
+               ARCH="aarch64"
+               MACHINE="virt"
+               ;;
+      *)
+               echo "invalid Arch, only support x86_64, ppc64le, aarch64"
+               exit 1
+               ;;
+esac
```

Comment:
@zhlhahaha the node-labeler code has been reverted recently (#3498), let's remove this change.